### ISSUE TITLE
JOYCON_GYRO_MASK = IGNORE_RIGHT disables R stick as mouse.

### ISSUE THREAD

**Issue by promocode2** [JibbSmart/JoyShockMapper#23]
hi love the work. thanks so much for taking the time to put this together. there's are a couple of projects up & running and this is the one for me.

I've got used to using gyro with my left hand. Given the option it feels most natural for me. I've used this method JOYCON_GYRO_MASK = IGNORE_RIGHT in various places in mouse and game profiles however it blocks using R analogue stick as mouse (makes it hard to react). 

The console still recognises RRING : trues and falses, as if the mouse was working, but no dice. The mouse does not move. 

I've tried to combine USE_BOTH but one just overrides the other. 

do you have any advice about this or an idea of a work around?

basically a big fan, many thanks. 
Rob

**Comment by JibbSmart** [comment]
Thanks very much :) That sounds like unintented behaviour to me. I'll check it out. Thanks for reporting it.

**Comment by JibbSmart** [comment]
I haven't had the time to look at this yet, but it's still on my radar. Sorry about the wait.

**Comment by promocode2** [comment]
It's okay. congrats on your new versions. mode shift sounds cool and can't wait to 'read me' about mouse area and other new bits. 

Just had a look to see if anything had changed in regards to my Q. IGNORE_RIGHT still kills the right mouse stick and the left gyro will only register vertically. IGNORE_BOTH will also kill the mouse stick, essentially blocking all mouse movement inputs.

Keep up the good work and once again, congrats on getting your flick stick onto steam :p

**Comment by promocode2** [comment]
in the meantime lol ive been using a £20 bootleg R joycon that incredibly registers as a pro controller on steam. they cheap so triggers wear down quickly (RIP bootleg L joycon) but i am still amazed with their flash xD.

so I've got JSM on L joycon and steam config on R bootleg joycon. beautiful gyro mess :D
